.github/dependabot.yml

```diff
@@ -1,8 +1,14 @@
 version: 2
-update_configs:
-  - package_manager: "java:maven"
+updates:
+  # Mvn dependencies
+  - package-ecosystem: "maven"
     directory: "/src/backend"
-    update_schedule: "daily"
-  - package_manager: "javascript"
+    schedule:
+      interval: "daily"
+      time: "18:00"
```

Comment:
6am not 6pm :P

Reply:
UTC ;-)

Reply:
it should run at 11 to test it today ;-) if you merge before ahahahhahahaahw

Reply:
SNEAKY